modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -112,27 +110,65 @@ function perflab_aea_enqueued_js_assets_test() {
 			'color' => 'blue',
 		),
 		'description' => sprintf(
-		/* translators: 1: Number of enqueued scripts. 2: "script" word. 3.Scripts size. */
-			'<p>' . esc_html__( 'The amount of %1$s enqueued %2$s (size: %3$s) is acceptable.', 'performance-lab' ) . '</p>',
-			$enqueued_scripts,
-			_n( 'script', 'scripts', $enqueued_scripts, 'performance-lab' ),
-			size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+			'<p>%s</p>',
+			esc_html(
+				sprintf(
+				/* translators: 1: Number of enqueued styles. 2.Styles size. */
+					_n(
+						'The amount of %1$s enqueued script (size: %2$s) is acceptable.',
+						'The amount of %1$s enqueued scripts (size: %2$s) is acceptable.',
+						$enqueued_scripts,
+						'performance-lab'
+					),
+					$enqueued_scripts,
+					size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+				)
+			)
 		),
 		'actions'     => '',
 		'test'        => 'enqueued_js_assets',
 	);
 
-	if ( $enqueued_scripts > 30 || perflab_aea_get_total_size_bytes_enqueued_scripts() > 300000 ) {
+	/**
+	 * Filters number of enqueued scripts to trigger warning.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param int $scripts_limit Scripts limit number. Default '30'.
```

Comment:
Nit-pick, but since the value is not a string, this would be more clear just as `Default 30.`